src/views/publications/PublicationList.vue

```diff
@@ -89,10 +89,10 @@ import { navigationStore, publicationStore } from '../../store/store.js'
 					:counter-number="publication?.attachmentCount.toString()"
 					@click="setActive(publication)">
 					<template #icon>
-						<ListBoxOutline v-if="publication.status === 'published'" :size="44" />
-						<ArchiveOutline v-if="publication.status === 'archived'" :size="44" />
-						<Pencil v-if="publication.status === 'concept'" :size="44" />
-						<AlertOutline v-if="publication.status === 'retracted'" :size="44" />
+						<ListBoxOutline v-if="publication.status.toLowerCase() === 'published'" :size="44" />
+						<ArchiveOutline v-if="publication.status.toLowerCase() === 'archived'" :size="44" />
+						<Pencil v-if="publication.status.toLowerCase() === 'concept'" :size="44" />
+						<AlertOutline v-if="publication.status.toLowerCase() === 'retracted'" :size="44" />
```

Comment:
According to the documentation, status has first capital, so making it lowercase seems redundant.
It would be better if the string it's compared to also has first capital.